### PR TITLE
Rename  field-content to fields-content

### PR DESCRIPTION
**Pull request by MadeByMike**
Renamed `@keystonejs/field-content` to `@keystonejs/fields-content`.

### CODE REVIEW

**Comment by MadeByMike** [comment]
I made a decision to keep the version number despite the rename. Since there were no functional change to the package I felt this makes sense.

File: .changeset/smart-baboons-shave.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+'@keystonejs/fields-content': major
```
Comment (MadeByMike) [inline .changeset/smart-baboons-shave.md:2]:
```suggestion
'@keystonejs/field-content': major
'@keystonejs/fields-content': major
```

File: .changeset/smart-baboons-shave.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+'@keystonejs/field-content': major
```
Comment (mitchellhamilton) [inline .changeset/smart-baboons-shave.md:2]: This can't be here given the package no longer exists
```suggestion
```

Comment (timleslie) [reply .changeset/smart-baboons-shave.md:2]: I think perhaps what we should do is to keep the original package, perform a `major` bump, and have that final package simply throw an error which points the dev towards the new package.

Once that final package has been published we can remove the package from the repo.

Thoughts?

Comment (mitchellhamilton) [reply .changeset/smart-baboons-shave.md:2]: Yep, that sounds good.

**Pull request merged**